src/value.h

```diff
@@ -146,18 +151,24 @@ void RSValue_Clear(RSValue *v);
  * the actual value object */
 void RSValue_Free(RSValue *v);
 
-
-RSValue* RSValue_IncrRef(RSValue* v);
-void RSValue_Decref(RSValue* v);
+RSValue *RSValue_IncrRef(RSValue *v);
+void RSValue_Decref(RSValue *v);
```

Comment:
```suggestion
void RSValue_DecRef(RSValue *v);
```

Consistent with `IncrRef`